configs/AM62AX/AM62AX_linux_toc.txt

```diff
@@ -23,12 +22,10 @@ linux/Overview_Technical_Support
 linux/Overview_GPLv3_Disclaimer
 linux/Overview/GCC_ToolChain
 linux/Overview/Processor_SDK_Linux_create_SD_card
-linux/Overview/Processor_SDK_Building_The_SDK
```

Comment:
same for this

Reply:
https://github.com/TexasInstruments/processor-sdk-doc/pull/132#discussion_r1889001624